js/scripts.js

```diff
@@ -0,0 +1,18 @@
+function drawTree(a) {
+  var i = 1;
+  for (i = 1; i <= a*2; i++) {
```

Comment:
unnecessary duplicates assign to iterator :) you can write it in loop `for (var i = 0;`